libs/server/store/index.ts

```diff
@@ -3,8 +3,8 @@ import { StoreS3 } from './providers/s3'
 
 export function createStore() {
   return new StoreS3({
-    accessKey: getEnv('STORE_ACCESS_KEY', undefined, true),
-    secretKey: getEnv('STORE_SECRET_KEY', undefined, true),
+    accessKey: getEnv('STORE_ACCESS_KEY'),
+    secretKey: getEnv('STORE_SECRET_KEY'),
```

Comment:
This uses the old config system, which is the main reason why this cannot be merged yet.